v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,229 @@
+    }[];
+}[]
+```
+
+Here you can see some examples of different types of brute force checks that can be performed:
+
+```typescript
+// Useful for limiting a user's attempt from the same network
+// This is the most common use case
+// ---
+// This does two check:
+// 1. 1 request per second - fast rate of requests
+// 2. 100 requests per 60 minutes - slow brute force - some attackers might try sidestepping the regular brute force detection by using a slower rate of requests
+const checkUserInSameNetwork = [{
+    keyStr: `${userIp}-${userEmail}`;
+    maxRequests: [
+        {
+            limit: 1;
+            perTimeIntervalMS: 1000; 
+        },
+        {
+            limit: 100;
+            perTimeIntervalMS: 60 * 1000 * 60; 
+        }
+    ];
+}]
+```
+
+
+```typescript
+// Useful for limiting requests from the same network
+// This should usually have a higher number of requests/time interval allowed
+const checkNetwork = [{
+    keyStr: `${userIp}`;
+    maxRequests: [
+        {
+            limit: 100;
+            perTimeIntervalMS: 1000; 
+        },
+    ];
+}]
+```
+
+```typescript
+// Useful for limiting requests for the user only
+const checkUserOnly = [{
+    keyStr: `${userEmail}`;
+    maxRequests: [
+        {
+            limit: 1;
+            perTimeIntervalMS: 1000; 
+        },
+    ];
+}]
+```
+
+```typescript
+// Checking by multiple keys at once
+const checkUserOnly = [
+    {
+        keyStr: `${userIp}-${userEmail}`;
+        maxRequests: [
+            {
+                limit: 1;
+                perTimeIntervalMS: 1000; 
+            },
+            {
+                limit: 100;
+                perTimeIntervalMS: 60 * 1000 * 60; 
+            }
+        ];
+    },
+    {
+        keyStr: `${userIp}`;
+        maxRequests: [
+            {
+                limit: 100;
+                perTimeIntervalMS: 1000; 
+            },
+        ];
+    }
+]
+```
+
+- `passwordHash`: The SHA-1 hash of the password that needs to be checked against the breach database. If this is not provided, the password breach check will be skipped.
+- `email`: The email address that is being used for the authentication event. If this is not provided (or `phoneNumber'), the impossible travel detection, new device detection and device count detection will be skipped.
+- `phoneNumber`: The phone number that is being used for the authentication event. If this is not provided (or `email'), the impossible travel detection, new device detection and device count detection will be skipped.
+- `actionType`: The type of action that is being performed. The possible values are:
+      - `emailpassword-sign-in`
+      - `emailpassword-sign-up`
+      - `send-password-reset-email`
+      - `passwordless-send-email`
+      - `passwordless-send-sms`
+      - `totp-verify-device`
+      - `totp-verify-totp`
+      - `thirdparty-login`
+      - `emailverification-send-email`
```

Comment:
Use `""` instead of backticks. Like "emailpassword-sign-in"

Reply:
done